File: ClientApp/src/app/model/interfaces/game.ts
import { Teams } from '../enums/teams';

export type Team = keyof typeof Teams;

export type PickValue = 0 | 0.5 | 1;

export interface Game {
  id?: number;
  week: number;
  homeTeam: Team;
  homeSpread: number;
  homeScore: number;
  homePickValue: PickValue;
  awayTeam: Team;
  awaySpread: number;
  awayScore: number;
  awayPickValue: PickValue;
  startTime: Date;
}

File: Models/Game.cs
namespace supercontestV2.Models
{
    public class Game
    {
        public int Id { get; set; }
        public int Week { get; set; }
        public string HomeTeam { get; set; }
        public int? HomeSpread { get; set; }
        public int? HomeScore { get; set; }
        public float? HomePickValue { get; set; }
        public string AwayTeam { get; set; }
        public int? AwaySpread { get; set; }
        public int? AwayScore { get; set; }
        public int? AwayPickValue { get; set; }

        public DateTime StartTime { get; set; }



    }
}